src/aks-preview/azcli-aks-live-test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:18.04
```

Comment:
do we still need this image definition?

Reply:
In case pull from mcr fails, we could build image locally

Reply:
> In case pull from mcr fails, we could build image locally

don't see any reason why pull from mcr would fail.
and also we will cache the image in our vsts agents.

Reply:
ok, removing Dockerfile and local build option in commit [47bb7f3](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/47bb7f3b0ff9ae58f87bebc472f84f2366a19207).